index.html

```diff
@@ -1465,8 +1465,12 @@ <h3 id="actions">Actions</h3>
         </section>
         <section>
           <h3 id="events">Events</h3>
-          <p>The supported protocols for events are
-            WebHooks, WebSockets, SSE and Long polling.</p>
+          <section class="ednote" Under discussion>
+            The candidate protocols for events are
```

Comment:
I think here should be: candidate *subprotocols*.

Reply:
+1